simplefs/simplefs.go

```diff
@@ -961,6 +980,42 @@ func (k *SimpleFS) SimpleFSCopy(ctx context.Context, arg keybase1.SimpleFSCopyAr
 		})
 }
 
+// SimpleFSSymlink starts making a symlink of a file or directory
+func (k *SimpleFS) SimpleFSSymlink(ctx context.Context, arg keybase1.SimpleFSSymlinkArg) (err error) {
+	// This is not async.
+	ctx, err = k.startSyncOp(ctx, "Symlink", arg)
+	if err != nil {
+		return err
+	}
+	defer func() { k.doneSyncOp(ctx, err) }()
+
+	// Get root FS of dst
+	t, tlfName, restOfDstPath, finalDstElem, err := remoteTlfAndPath(arg.Dest)
```

Comment:
Instead of this block, from here down to `NewFS()`, you should be able to use `k.getFS()`, like in `SetStat()`.  Though you might want to add a check that the dest path type is exactly `KBFS` first.